fluentd-elasticsearch/OWNERS

```diff
@@ -0,0 +1,10 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+- coffeepac
+- monotek
+- sig-instrumentation-approvers
+reviewers:
+- sig-instrumentation-reviewers
```

Comment:
You will need to define an OWNERS_ALIASES for these in the root of the repo for this to work.

See e.g. https://github.com/kubernetes-sigs/prometheus-adapter/blob/master/OWNERS_ALIASES

Reply:
I think we'll need https://github.com/kubernetes-sigs/instrumentation-addons/pull/3 approved/merged first.  then we can run `/verify-owners` here and it'll get sorted.

Reply:
If added the OWNERS_ALIAS file here too, as in https://github.com/kubernetes-sigs/instrumentation-addons/pull/2

If we only need it in the repo rootm, as suggested by @ehashman, i'll remove it again.